target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -123,6 +123,23 @@ define Device/tplink_tl-wr1043nd-v1
 endef
 TARGET_DEVICES += tplink_tl-wr1043nd-v1
 
+define Device/tplink_tl-wr810n-v1
+  $(Device/tplink-8mlzma)
+  ATH_SOC := qca9531
+  DEVICE_TITLE := TP-LINK TL-WR810N v1
+  TPLINK_HWID := 0x8100001
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport
```

Comment:
I'm quite sure kmod-usb-core can be dropped here. It should be autoselected as dependency anyway.

Reply:
Copy'n'pasted, but is fixed now :-)